CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 - Env variable `BUSBAR_COMPONENT` to tell the component type from the Procfile
 - Env variable `BUSBAR_NODETYPE` to show the currently set nodetype
 
+## Update
```

Comment:
`Update` is not a valid choice as seen on [keepachangelog.org](https://keepachangelog.com/en/1.0.0/):
![valid-changes](https://user-images.githubusercontent.com/183227/61912771-dd9d4880-aeef-11e9-9851-77e0523a6e16.png)

In my opinion, bumping the Gemfile does not warrant a `CHANGELOG` entry. But if you insist, please choose a different type of change and be sure to keep it in the past tense.